### PR TITLE
refactor(): add common cron patterns

### PR DESCRIPTION
**Pull request by jeffminsungkim**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## What is the current behavior?
```typescript
import { Injectable, Logger } from '@nestjs/common';
import { Cron } from '@nestjs/schedule';

@Injectable()
export class TasksService {
  private readonly logger = new Logger(TasksService.name);

  @Cron('45 * * * * *')
  handleCron() {
    this.logger.debug('Called when the current second is 45');
  }
}
```

## What is the new behavior?
```typescript
import { Injectable, Logger } from '@nestjs/common';
import { Cron, TaskSchedule } from '@nestjs/schedule';

@Injectable()
export class TasksService {
  private readonly logger = new Logger(TasksService.name);

  @Cron(TaskSchedule.EVERY_30_SECONDS)
  handleCron() {
    this.logger.debug('Call every 30 seconds');
  }
}
```

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

**Comment by jeffminsungkim** [comment]
#23

I'll try to update the docs and add more tests if necessary.

File: lib/enums/task-schedule.enum.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
+export enum TaskSchedule {
```
Comment (kamilmysliwiec) [inline lib/enums/task-schedule.enum.ts]: What about the `CronExpression` name instead?

**Comment by kamilmysliwiec** [comment]
Thanks for your contribution. I left 1 comment

File: lib/enums/task-schedule.enum.ts (hunk 1 above)
Comment (jeffminsungkim) [reply lib/enums/task-schedule.enum.ts]: Sounds good!

**Comment by kamilmysliwiec** [comment]
Thank you @jeffminsungkim! Can you please create a PR to the docs? A short note about this fetaure should be enough

**Comment by jeffminsungkim** [comment]
My pleasure. @kamilmysliwiec  I'll try to make one 😬 😳 (This is my first time..)

**Comment by jeffminsungkim** [comment]
@kamilmysliwiec I've updated the documentation.

**Comment by kamilmysliwiec** [comment]
Can you please resolve conflicts? :) Thank you!

**Comment by jeffminsungkim** [comment]
@kamilmysliwiec Done 😃

**Comment by kamilmysliwiec** [comment]
Thank you!

**Pull request merged**